pkg/api/api.proto

```diff
@@ -33,6 +33,7 @@ message GetProductSummaryResponse {
 message ProductSummary {
   string app = 1;
   string version = 2;
+  string link_version = 3;
```

Comment:
Why not just put the source commit id and display version here just like we do with the normal releases? Then you wouldn't need to duplicate the logic in the backend.

Reply:
I was just trying to keep the logic out of the frontend because based on this story https://revolution.dev/app/-JqFGExX46gs9mH7vxR5/-NQPLdkfXUNc4bE9KOiO/STORY/-NdArQWgtNTXtGiJFTVH/ there is really only one value to display. If it makes sense I can add all values but also its just a preference if this logic would be in the front end or the backend

Reply:
It's a bit more than a preference. Currently this logic is only in the frontend ( see https://github.com/freiheit-com/kuberpult/blob/ce320ba275b3902d0cc3e42afc369025535a843f/services/frontend-service/src/ui/components/ReleaseVersion/ReleaseVersion.tsx#L31 ). It's not too bad to duplicate this, but I think it will make your life easier to not do that and put it in the frontend in both cases.

Reply:
fixed and now all logic is dealt with in the front end